snapfaas/src/sched/messages.proto

```diff
@@ -52,11 +46,6 @@ message ProcessTask {
     LabeledInvoke labeledInvoke = 2;
 }
 
-message ProcessTaskInsecure {
-    string taskId = 1;
-    UnlabeledInvoke unlabeledInvoke = 2;
-}
```

Comment:
Why remove ProcessTaskInsecure and UnlabeledInvoke?

Reply:
It was there because I thought we needed it to measure an insecure version's performance. But in fact, we just needed to comment out certain lines to have an insecure version.